app/controllers/application_controller.rb

```diff
@@ -1,2 +1,20 @@
+# frozen_string_literal: true
+
 class ApplicationController < ActionController::Base
+  helper_method :current_user, :logged_in?
```

Comment:
If these two methods are being used in your views, then I'd recommend moving them to your `users_helper.rb`, which is what our requirements guidelines recommend also.

[`OPTIONAL`]